docs/help/contributor/style-guide/style-and-formatting.md

```diff
@@ -1,11 +1,11 @@
 # Formatting standard and conventions
 
 ## Documenting Knative API Objects
-> Any Knative API Objects ought to use Title case.
+>  Use title case for Knative API Objects.
```

Comment:
Should this be camel case (specifically upper camel case) for API objects, instead of title case? For example, ContainerSource.